packages/charts/src/renderers/xy-renderer.ts

```diff
@@ -18,5 +28,18 @@ export abstract class XYRenderer<TA extends IAccessors> extends Renderer<TA> {
             y: scales.y.convert(dataSeries.accessors.data.y(point, index, dataSeries.data, dataSeries)),
         };
     }
+    public getDataPointIndex(series: IDataSeries<IXYAccessors>, values: { [p: string]: any }, scales: Scales): number {
+        if (isUndefined(values.x)) {
+            return DATA_POINT_INTERACTION_RESET;
+        }
 
+        // @ts-ignore
```

Comment:
Not sure why I needed this ts-ignore but it works with it I would like a suggestion on how to fix this if possible